.github/workflows/security-scan.yaml

```diff
@@ -0,0 +1,80 @@
+name: Security scan
+
+permissions:
+  contents: read  # for actions/checkout to fetch code
+  security-events: write  # for github/codeql-action/upload-sarif to upload SARIF results
+
+on:
+  workflow_call:
+    inputs:
+      # Download k8s-snap using either a GH action artifact or a snap channel.
+      artifact:
+        description: The name of a GH action artifact.
+        type: string
+      channel:
+        description: k8s snap channel.
+        type: string
+      checkout-ref:
+        description: k8s-snap git checkout ref, optional.
+        type: string
+
+jobs:
+  get-tags:
```

Comment:
I guess this should be called something like `security-scan`?

Reply:
Whops, indeed.